server/__test__/AccountsService.test.js

```diff
@@ -0,0 +1,171 @@
+const {
+    createNewAccount,
+    accountDoesExist,
+    validateFields,
+    containsSpecialCharacters
+} = require('../src/service/AccountsService');
+const accountsDao = require('../src/repository/AccountsDAO');
+
+
+describe('createNewAccount Tests', () => {
+
+    test('should return data successfully', async () => {
+        const receivedData = {
+            username: 'test_username',
+            password: 'password123',
+            isAdmin: false
+        };
+        jest.spyOn(accountsDao, 'getAccountByUsername').mockReturnValueOnce({});
+        jest.spyOn(accountsDao, 'createNewAccount').mockReturnValueOnce({
```

Comment:
DynamoDB does not return the new entry after a `PutCommand`.  You didn't need to return an item.